Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```diff
@@ -36,3 +36,46 @@
   - type: Appearance
     visuals:
     - type: EnergySwordVisualizer
+
+- type: entity
+  name: pen
+  parent: EnergySword
+  id: EnergyDagger
+  suffix: E-Dagger
+  description: 'A dark ink pen.'
+  components:
+  - type: EnergySword
+    litDamageBonus:
+        types:
+            Slash: 6
+            Heat: 6
```

Comment:
I would buff this to 18 damage (9+9), currently it does less damage then both the butchers cleaver and the survival knife which are also pretty concealable and obtainable and are not traitor items.

Maybe buff the esword to 30 from 25 if you think that needs more value.

Reply:
> Accidental edit to `e_sword.rsi`?

aww shoot, took me a minute to see that, yeah I was messing with my compression settings in Gimp to try to figure out why the hell I was up around 10-15kb per file vs around 200bytes, i'll reverse that in a sec